ydb/core/tx/datashard/export_common.cpp

```diff
@@ -71,6 +71,27 @@ TMaybe<Ydb::Table::CreateTableRequest> GenYdbScheme(
     return scheme;
 }
 
+TMaybe<Ydb::Scheme::ModifyPermissionsRequest> GenYdbPermissions(
+        const NKikimrSchemeOp::TPathDescription& pathDesc)
+{
```

Comment:
Вроде и в одну строку не слишком длинно получается:
```suggestion
TMaybe<Ydb::Scheme::ModifyPermissionsRequest> GenYdbPermissions(const NKikimrSchemeOp::TPathDescription& pathDesc) {
```